src/components/pages/AccountPage.js

```diff
@@ -0,0 +1,25 @@
+import { React, useContext } from "react";
+import Navbar from "../NavBar";
+import Sidebar from "../SideBar";
+import { UserContext } from "../../constants/UserContext";
+
+const AccountPage = () => {
+  const { userInfo } = useContext(UserContext);
+
+  return (
+    <div className="w-full h-screen bg-dark-gray">
+      <Navbar />
+      <Sidebar />
+      <h1 className="font-semibold text-5xl text-white text-center pt-32">
+        Account Page
+      </h1>
+      <p className="text-white text-center">
+        {userInfo?.attributes?.given_name +
```

Comment:
Both `given_name` and `family_name` are undefined when I click on the account page.

Reply:
Yeah for some reason it only updates upon refresh. Trying to find a workaround for this.